pkg/datarepair/repairer/repairer.go

```diff
@@ -25,7 +26,7 @@ type Repairer interface {
 
 // Config contains configurable values for repairer
 type Config struct {
-	QueueAddress string        `help:"data repair queue address" default:"redis://localhost:6379?db=5&password=123"`
+	QueueAddress string        `help:"data repair queue address" default:"redis://localhost:6379?db=5&password=abc123"`
```

Comment:
Do not default the global config to test queue.

Reply:
The default doesn't match the test queue. I just happened to leave the password the same but the ports are different and the db number is different

Reply:
Ah right.